core/src/pretty.rs

```diff
@@ -42,6 +42,13 @@ fn sorted_map<K: Ord, V>(m: &'_ IndexMap<K, V>) -> Vec<(&'_ K, &'_ V)> {
     ret
 }
 
+pub fn escape(s: &str) -> String {
```

Comment:
Nitpick, but shouldn't this function be moved to `indentifier` or even to `term::string` if it's used inside `identifier`? it's not obvious that `identifier` should be depending on `pretty`

Reply:
That's a good point. It felt too unrelated to `Ident` to go into `identifier` to me. In fact, `Ident::label_quoted` is used only in the pretty printer. So maybe the correct course of action to remove this dependency would be to move `label_quoted` into `pretty`? I'm not sure.

Reply:
Hmm, good point as well. I think both are reasonable:

- Leave `label_quoted` in identifier and bring back escape either in `identifier` or `term::string`, because I believe what you're doing in the end is to escape a Nickel string. It would like implementing `Display`: I think it's fair to have it inside identifier (or, should we just implement `Display`, by the way, instead of `label_quoted`?)
- Move both to `pretty`, because it's about pretty-printing stuff

I guess the current in-between with one inside `identifier` and one inside `pretty` is what's feeling itchy